src/Parser/Rogue/Assassination/Modules/Legendaries/DuskwalkersFootpads.js

```diff
@@ -35,13 +35,12 @@ class DuskwalkersFootpads extends Analyzer {
     }
 
     const spent = event.resourceChange;
+    const reduction = spent * VENDETTA_CDR_PER_ENERGY;
 
     if (!this.spellUsable.isOnCooldown(SPELLS.VENDETTA.id)) {
-      this.wastedReduction += spent * VENDETTA_CDR_PER_ENERGY;
+      this.wastedReduction += reduction;
     } else {
 
-      const reduction = spent * VENDETTA_CDR_PER_ENERGY;
-
       this.spellUsable.reduceCooldown(SPELLS.VENDETTA.id, reduction * 1000);
```

Comment:
The return value of this is the effective reduction (if you're at 2 sec cd remaining and reduce it by 3 sec this would return 2), so you could add this value instead of `reduction` to `this.wastedReduction` for a slightly more accurate result.